tools/code-analyzer/README.md

```diff
@@ -12,17 +12,17 @@ Currently there are just 2 commands:
 
 Here is an example `analyzer` command:
 
-`pnpm run analyzer -- lint "release/6.8" "6.8.0" -b=release/6.7`
+`pnpm run analyzer lint -- "release/6.8" "6.8.0" -b=release/6.7`
```

Comment:
These commands did work, but don't need to `--` the command name so I've improved the instructions